src/components/scenes/WaitScene.tsx

```diff
@@ -0,0 +1,50 @@
+import LottieView from 'lottie-react-native'
+import * as React from 'react'
+import { View } from 'react-native'
+
+import { RouteProp } from '../../types/routerTypes'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+type Props = {
+  message: string
+  route: RouteProp<'waitScene'>
+}
+
+export const WaitScene = (props: Props) => {
+  const { message } = props.route.params
+
+  const theme = useTheme()
+  const styles = getStyles(theme)
```

Comment:
We have too many of these scenes already. We have `LoadingScene`, `FullScreenLoader`, and `AirshipFullScreenSpinner`, at least. There is really no need for another one of these. If we really care about animations, maybe we can update one of the other scenes. However, Lottie seems like a pretty heavy dependency to take on.